src/main/resources/assets/tfc/lang/en_us.lang

```diff
@@ -808,7 +808,7 @@ config.tfc.animals.donkey=Donkey
 config.tfc.animals.mule=Mule
 config.tfc.animals.ocelot=Ocelot
 config.tfc.animals.wolf=Wolf
-config.tfc.animals.bear=Bear
+config.tfc.animals.bear=Grizzly Bear
```

Comment:
It seems that this should be `config.tfc.animals.grizzly_bear`

Reply:
updated